### PR TITLE
 Create a unique launch agent for each architecture

### PR DESCRIPTION
**Pull request by lkrms**
> Apologies for the double-up; #62 was based on my `master` branch and I was hoping the PR would follow when I renamed it while rebasing on the upstream `master`. Lesson learned: **always** branch before opening a PR! The original description follows.

On an Apple Silicon system with 2 Homebrew installations (at `/opt/homebrew` and `/usr/local` for arm64 and x86_64 respectively), `arch --x86_64 /usr/local/bin/brew autoupdate subcommand` conflicts with `/opt/homebrew/bin/brew autoupdate subcommand`, which makes it impossible to enable autoupdate for both installations simultaneously.

The workaround I've used here may not be suitable for wider adoption, but hopefully a few revisions will get it there if not.

TL;DR:

- the LaunchAgent's name now includes the applicable architecture (e.g. `com.github.domt4.homebrew-autoupdate.arm64`)
- the previous name is kept if `~/Library/LaunchAgents/com.github.domt4.homebrew-autoupdate.plist` already exists (the new name only takes effect after `brew autoupdate delete`)
- `~/Library/Logs/com.github.domt4.homebrew-autoupdate/` isn't deleted unless it's empty

There are probably things I've missed, but it seems to be working as expected here (although I don't have `--upgrade` enabled).

### CODE REVIEW

**Comment by lkrms** [comment]
FYI @DomT4, I've just rebased and cleaned up this branch, with a couple of improvements:

- the notifier now reads from the log file associated with the instance raising the notification (NB: the updated app is currently unsigned), and
- at runtime, `brew update` etc. are now invoked using the same architecture `brew autoupdate` was originally called with

The changes haven't come through because this PR is closed, but I assume you can remedy that? Let me know if you'd like me to open another PR. You can check out the branch [here](https://github.com/lkrms/homebrew-autoupdate/commits/support_multiple_installs) if needed.
